### PR TITLE
Rewrite of log-capture, print and assert for tests

### PR DESCRIPTION
**Pull request by ahaarrestad**
Utilizing ShadowLog which is part of Robolectric

### CODE REVIEW

**Comment by ahaarrestad** [comment]
@jamorham is this what you were looking for regarding logs in tests?

File: app/src/test/java/com/eveningoutpost/dexdrip/RobolectricTestWithConfigAndPowerMockLogRedirect.java [hunk 1]
```diff
@@ -1,112 +0,0 @@
-package com.eveningoutpost.dexdrip;
```
Comment (jamorham) [inline app/src/test/java/com/eveningoutpost/dexdrip/RobolectricTestWithConfigAndPowerMockLogRedirect.java]: Can you undelete this file. It is a useful example I think of how to get powermock and roboelectric actually working together. We don't need to use it but I don't think it should be deleted.

**Comment by ahaarrestad** [comment]
Sure, but why would you need powermock? You can use robolectric shadows...

Asbjørn

Den tir. 30. okt. 2018, 21:52 skrev JamOrHam <notifications@github.com>:

**Comment by jamorham** [comment]
Can RoboElectric shadows mock any static method?

**Comment by ahaarrestad** [comment]
I believe so: http://robolectric.org/extending/

**Comment by ahaarrestad** [comment]
Will change the commit (restore the file) as soon as possible.

**Review by jamorham — APPROVED** [review]
Thanks!

**Pull request merged**